app/javascript/src/components/Dashboard/Contacts/Form/constants.js

```diff
@@ -0,0 +1,27 @@
+import { t } from "i18next";
+import * as yup from "yup";
+
+export const INITIAL_FORM_VALUES = {
+  firstName: "",
+  lastName: "",
+  email: "",
+  role: "",
+};
+
+export const VALIDATION_SCHEMA = yup.object({
+  firstName: yup
+    .string()
+    .trim()
+    .required(t("common.errors.required", { field: "First Name" })),
+  lastName: yup
+    .string()
+    .trim()
+    .required(t("common.errors.required", { field: "Last Name" })),
+  email: yup
+    .string()
+    .email(t("common.errors.invalidEmail"))
+    .required(t("common.errors.required", { field: "Email" })),
+  role: yup.object().required(t("common.errors.required", { field: "Role" })),
```

Comment:
Shape of the object should be explicitly mentioned unless you have no knowledge of the shape.